SGWE consists of a GUI and a pipeline of scripts for the simulation of genome/proteome-wide evolution under heterogeneous substitution models and a variety of evolutionary scenarios.

The evolutionary history can be the same or change across regions. It can be either simulated by the coalescent (modified with demographics, recombination, populations/species tree and migration) or just user-specified by input tree/s.

Then, each genomic region can be simulated according to a particular substitution model of evolution based on a variety of user-specified distributions. A wide set of parametrical and empirical nucleotide, codon and amino acid substitution models are implemented. For example, SGWE can simulate genome evolution with both coding and non-coding regions, only non-coding regions, only coding regions or only amino acid regions.

`SGWE` requires `CoalEvol` (http://code.google.com/p/coalevol/) and `INDELible` (http://abacus.gene.ucl.ac.uk/software/indelible/).
A user-friendly GUI is included with the package.


#### Download ####
Details to download SGWE can be found at Wiki/Download\_SGWE (http://code.google.com/p/sgwe-project/wiki/Download_SGWE).


#### Citation ####
[Arenas M, Posada D. 2014. Simulation of Genome-wide Evolution under Heterogeneous Substitution models and Complex Multispecies Coalescent Histories. Molecular Biology and Evolution 31 (5): 1295-1301. ](http://mbe.oxfordjournals.org/content/31/5/1295.full)


#### Help ####
Please, do not hesitate to contact us (miguelmmmab@gmail.com) for any question.